lib/ecto/adapters/exqlite.ex

```diff
@@ -79,11 +79,6 @@ defmodule Ecto.Adapters.Exqlite do
     [&Codec.bool_decode/1, type]
   end
 
-  @impl Ecto.Adapter
-  def loaders(:binary_id, type) do
```

Comment:
Is this no longer necessary to specify? or was this causing an unrelated bug?

Reply:
Without this removed, I was hitting a `Jason` encoding error related to the UUID being provided as a (not sure of terminology) charlist. E.g.,
```
<<41, 220, 200, 189, 24, 33, 72, 249, 188, 18, 214, 5, 143, 148, 203, 28>>
```

Instead of, say "407ef7a9-0fd9-4abf-a5bc-22b7c12f0884"

Reply:
Oh I see. Let's remove it then and make a follow on issue to ensure that UUID is functioning as advertised and what not.